src/clib/lib/config/config_parser.cpp

```diff
@@ -663,12 +663,3 @@ void config_install_message(config_parser_type *config, const char *kw,
     hash_insert_hash_owned_ref(config->messages, kw,
                                util_alloc_string_copy(message), free);
 }
-
-void config_parser_deprecate(config_parser_type *config, const char *kw,
```

Comment:
There is still code related to this present;
config_parser.hpp
```
void config_parser_deprecate(config_parser_type *config, const char *kw,
                             const char *msg);
```
Should be removed also.

Reply:
ahh very good, of course, the header file. i didn't even grep for the function... thanks again ^^

Reply:
it is done.